sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```
@@ -100,6 +100,55 @@ instances:
     #     tags:
     #     - test:<INTEGRATION>
 
+    ## @param use_tls - boolean - optional - default: false
+    ## This instructs the SAP HANA integration to connect using TLS.
+    #
+    # use_tls: false
+
+    ## @param tls_verify - boolean - optional - default: true
+    ## Instructs the check to validate the TLS certificate of services.
```

```suggestion
    ## Instructs the check to validate the TLS certificates of services.
```
I think? multiple services will have multiple certificates?

---

True, will have to be in another PR though as this option is used in many integrations. It has to be changed for all integrations at once in our "specs"